webrtc-stats.html

```diff
@@ -2989,6 +2989,17 @@ <h2>
                   addresses, and fully qualified domain names (FQDNs). See
                   [[!RFC5245]] section 15.1 for details.
                 </p>
+                <div class="note">
```

Comment:
Don't use "note" for a normative statement - this paragraph has a MUST NOT. Just use <p>.